### PR TITLE
release: initial release notes

### PR DESCRIPTION
**Pull request by miekg**
Core of the release notes.

### CODE REVIEW

File: notes/coredns-1.7.0.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+In this release we have deprecated the *federation* plugin that was used in conjunction with the
+*kubernetes* plugin.
+
+Futher more a slew a spelling corrections and other minor improvements and polish. **And** three(!)
+new plugins.
+
+# Plugins
+
+* [*acl*](/plugins/acl) blocks queries depending on their source IP address.
+* [*googlecloud*](/plugin/googlecloud) to enabel serving zone data from GCP Cloud DNS.
```
Comment (stp-ip) [inline notes/coredns-1.7.0.md]:
```suggestion
* [*googlecloud*](/plugin/googlecloud) to enable serving zone data from GCP Cloud DNS.
```

**Review by stp-ip — APPROVED** [review]
/lgtm besides typo

File: notes/coredns-1.7.0.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
++++
+title = "CoreDNS-1.7.9 Release"
```
Comment (rdrozhdzh) [inline notes/coredns-1.7.0.md]: 1.7.0

File: notes/coredns-1.7.0.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+In this release we have deprecated the *federation* plugin that was used in conjunction with the
+*kubernetes* plugin.
+
+Futher more a slew a spelling corrections and other minor improvements and polish. **And** three(!)
+new plugins.
+
+# Plugins
+
+* [*acl*](/plugins/acl) blocks queries depending on their source IP address.
+* [*googlecloud*](/plugin/googlecloud) to enabel serving zone data from GCP Cloud DNS.
```
Comment (johnbelamaric) [inline notes/coredns-1.7.0.md]: s/googlecloud/clouddns/g

File: notes/coredns-1.7.0.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+In this release we have deprecated the *federation* plugin that was used in conjunction with the
+*kubernetes* plugin.
+
+Futher more a slew a spelling corrections and other minor improvements and polish. **And** three(!)
+new plugins.
+
+# Plugins
+
+* [*acl*](/plugins/acl) blocks queries depending on their source IP address.
+* [*clouddns*](/plugin/clouddns) to enabel serving zone data from GCP Cloud DNS.
```
Comment (mrueg) [inline notes/coredns-1.7.0.md]: *enable

**Pull request merged**